### ISSUE TITLE
Size of num_gpu while training AVmodel?

### ISSUE THREAD

**Issue by 83344rushikesh** [JusperLee/Looking-to-Listen-at-the-Cocktail-Party#5]
??

**Comment by JusperLee** [comment]
eight gpus for training. However, due to the large amount of data, the training speed is very slow.  It is recommended that you use more GPUs for training.

**Comment by 83344rushikesh** [comment]
@JusperLee okay.. What should be done if the keras is giving the loss function error?

**Comment by JusperLee** [comment]
Try changing line 11 from model_loss import audio_discriminate_loss2 as audio_loss in AV_train.py to model_loss import audio_discriminate_loss as audio_loss.

**Comment by 83344rushikesh** [comment]
@JusperLee I have done that still it is coming
